ASTNode.h

```diff
@@ -550,13 +550,14 @@ class ASTCondBlock : public ASTBlock {
 
 class ASTIterBlock : public ASTBlock {
 public:
-    ASTIterBlock(ASTBlock::BlkType blktype, int end, PycRef<ASTNode> iter)
-        : ASTBlock(blktype, end), m_iter(std::move(iter)), m_idx(), m_comp() { }
+    ASTIterBlock(ASTBlock::BlkType blktype, int start, int end, PycRef<ASTNode> iter)
```

Comment:
Save the position of the for loop and use it to find block extents. Later, continue statements could be inferred